library/spdm_responder_lib/libspdm_rsp_csr.c

```diff
@@ -141,6 +139,13 @@ libspdm_return_t libspdm_get_response_csr(libspdm_context_t *spdm_context,
             response_size, response);
     }
 
+    opaque_data = (void*)(requester_info + requester_info_length);
+    if(opaque_data == NULL) {
```

Comment:
How can `opaque_data` be `NULL`?

Reply:
Thanks. 
`opaque_data `will not be `NULL`,  I have deleted it